addons/io/org.openhab.io.hueemulation/OSGI-INF/org.openhab.io.hueemulation.internal.HueEmulationServlet.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
looks as if you are missing a .gitignore file here

Reply:
fixed